File: Atividades em C/Atividade 4.c
#include <stdio.h>

int main()
{
    /*
    Atividade 4 | CLASSIFICAR TRIANGULO (LADOS)
    Indicar o tipo de triangulo comparando o tamanho dos lados fornecido pelo usuario
    */
    
    float A, B, C;

    printf("\tTIPOS DE TRIANGULOS | ANALISANDO A MEDIDA DOS LADOS\n\nDigite um numero positivo e aperte ENTER, ao inserir cada lado do triangulo:\n\n");
    
    printf("Lado A = ");
    scanf ("%f", &A);

    printf("Lado B = ");
    scanf ("%f", &B);

    printf("Lado C = ");
    scanf ("%f", &C);
  
    if((A+B>=C) && (A+C>=B) && (B+C>=A)){
        printf("\nTipo do triangulo: ");
        if((A==B) && (B==C)){
            printf("Equilatero (Todos os lados iguais)");
        }
        else if((A!=B) && (A!=C) && (B!=C)){
            printf("Escaleno (Todos os lados diferentes)");
        }
        else printf("Isosceles (2 lados iguais)");
    }
    else printf ("\nEssas medidas nao formam um triangulo.");
    
    return 0;
}

File: Atividades em Python/Atividade 4.py
"""
Atividade 4 | CLASSIFICAR TRIANGULO (LADOS)
Indicar o tipo de triangulo comparando o tamanho dos lados fornecido pelo usuario
"""
print("\n\tTIPOS DE TRIANGULOS | COMPARANDO A MEDIDA DOS LADOS\n\nDigite um numero positivo e aperte ENTER, ao inserir cada lado do triangulo:\n")

A = int(input("Lado A = "))
B = int(input("Lado B = "))
C = int(input("Lado C = "))

if A + B >= C and A + C >= B and B + C >= A and A > 0 and B > 0 and C > 0:
    print("\nTipo do triangulo: ")
    if A == B and B == C:
        print("Equilatero (Todos os lados iguais)") 
    elif A != B and A != C and B != C:
        print("Escaleno (Todos os lados diferentes)")
    else: print("Isosceles (2 lados iguais)")       #if A == B or A == C or B == C: print("Isosceles") | Condicao que poderia ser usada.
else: print ("\nEssas medidas nao formam um triangulo.")